File: ITMO.Python/PythonProject/Demo02/benchmark/csharp/csharp/Program.cs
using csharp;

Console.WriteLine("[C#] Введите путь к входному файлу => ");
var inputString = Console.ReadLine().Split();
string inputPath = inputString[0];
string dataType = inputString[1];
Console.WriteLine($"[C#] Путь к входному файлу: {inputPath}");

Starter starter = new Starter(inputPath, dataType);
Console.WriteLine($"[C#] {await starter.StartBenchmark()}");

File: ITMO.Python/PythonProject/Demo02/benchmark/java/src/Program.java
import java.util.Scanner;

public class Program {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("[Java] Введите путь к входному файлу => ");
        String[] inputString = scanner.nextLine().split(" ");
        scanner.close();

        String inputPath = inputString[0];
        String dataType = inputString[1];

        System.out.println("[Java] Путь к входному файлу: " + inputPath);
        Starter starter = new Starter(inputPath, dataType);
        System.out.println("[Java] " + starter.startBenchmark());

//        PrintStream printStream = new PrintStream(System.out, true, "utf-8");
//        printStream.println(stringPath);
    }
}
